src/staticanalysis/frontend/default.nix

```diff
@@ -0,0 +1,7 @@
+{ releng_pkgs
+}:
+
+releng_pkgs.lib.mkYarnFrontend {
+  src = ./.;
+	src_path = ./.;
```

Comment:
`src_path` should be "src/staticanalysis/frontend". This is a relative path used to `cd` in some scripts.

Reply:
Unfortunately that does not work, i get:

```
error: getting attributes of path '/app/src/staticanalysis/frontend/src/staticanalysis/frontend': No such file or directory
```

The two options working right now are `./.` or `../../../src/staticanalysis/frontend` (to be explicit)

I guess this issue will be solved when we switch to per-project config